Automation to update the input select widget

spaetz opened the pull request:
## Proposed change
In order to also update the input_select proposed in these docs in case some external event changes
the Vallox profile (web interface, mechanical switch, reboot, etc...) you
can use the following automation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
Added an automation that achieves that to the sample docs.

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

spaetz committed:
Automation to update the input select widget

In order to also update the input select in case some external event changes
the Vallox profile (web interface, mechanical switch, reboot, etc...) you
can use the following automation.

Update source/_integrations/vallox.markdown
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

frenck commented on source/_integrations/vallox.markdown:
````diff
@@ -67,6 +67,23 @@ automation:
+      option: '{{ states(''sensor.vallox_current_profile'') }}'
+    service: input_select.select_option
+```
+{% endraw %}
````
Add a new line at the end.

```suggestion
{% endraw %}

```

frenck commented on source/_integrations/vallox.markdown:
````diff
@@ -67,6 +67,23 @@ automation:
+```yaml
+automation:
+ - alias: Update vallox input_select
+  description: Update input_select when external event changes the profile
````
This YAML is actually invalid, as it is wrongly indented.

spaetz replied:
OK, should be valid now. I am a yaml noob to be honest, so thanks for the feedback.

frenck commented on source/_integrations/vallox.markdown:
````diff
@@ -67,6 +67,23 @@ automation:
+{% raw %}
+```yaml
+automation:
+ - alias: Update vallox input_select
````
```suggestion
 - alias: Update Vallox input_select
```

frenck commented on source/_integrations/vallox.markdown:
```diff
@@ -67,6 +67,23 @@ automation:
+  action:
+  - data_template:
+      entity_id: input_select.ventilation_profile
+      option: '{{ states(''sensor.vallox_current_profile'') }}'
```
```suggestion
      option: "{{ states('sensor.vallox_current_profile') }}"
```

frenck reviewed (changes requested):
Please do not remove the PR template. Update your PR description to have the PR template presented when opening a PR.

spaetz commented:
OK, addressed review comments. Thanks for the extreme quick feedback. Much appreciated. Unfortunately, I cannot update the PR description after I have created it, sorry we need to keep it as is, or I need to create a new PR.

P.S. I wonder why the page is saying "Valloxs" when the firm and the ventialtion units are clearly called "Vallox". Anyway, thanks for considering this mere.

frenck committed:
:pencil2: Tweak

frenck reviewed (approved):
Thanks, @spaetz 👍
